<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveOptionDialog</class>
 <widget class="QDialog" name="CurveOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curve Option Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMinPoints">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min points:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max error ratio to radius:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMaxErrorRatioToRadius">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cluster epsilon:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClusterEpsilon">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min angle:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMinAngle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min radius:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMinRadius">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max radius:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMaxRadius">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># iterations:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumIterations">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[deg]</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>lineEditNumIterations</tabstop>
  <tabstop>lineEditMinPoints</tabstop>
  <tabstop>lineEditMaxErrorRatioToRadius</tabstop>
  <tabstop>lineEditClusterEpsilon</tabstop>
  <tabstop>lineEditMinAngle</tabstop>
  <tabstop>lineEditMinRadius</tabstop>
  <tabstop>lineEditMaxRadius</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
